sdk/loadtesting/azure-developer-loadtesting/azure/developer/loadtesting/_generated/aio/operations/_patch.py

```diff
@@ -178,7 +141,7 @@ async def begin_test_run(
         old_test_run_id: Optional[str] = None,
         content_type: str = "application/merge-patch+json",
         **kwargs: Any
-    ) -> AsyncLoadTestingLROPoller[JSON]:
+    ) -> AsyncLROPoller[JSON]:
```

Comment:
Could we please delete the two `@overloads` that are being auto-generated on the `_operations.py` file for this `begin_test_run`?
Those overloads are still showing up in the APIview...